actionmailer/CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 ## Rails 4.0.0 (unreleased) ##
 
-*   Eager loading made to use relation's in_clause_length instead of host's one.
+*   Allow passing interpolations to `#default_i18n_subject`, e.g.:
+
+        # config/locales/en.yml
+        en:
+          user_mailer:
+            welcome:
+              subject: 'Hello, %{username}'
+
+        # app/mailers/user_mailer.rb
+        class UserMailer < ActionMailer::Base
+          def welcome(user)
+            mail(subject: default_i18n_subject(username: user.name))
+          end
+        end
```

Comment:
@rafaelfranca Why don't we have such examples mentioned in the [official doc](http://api.rubyonrails.org/classes/ActionMailer/Base.html#method-i-default_i18n_subject) ?